src/main/java/net/dv8tion/jda/api/interactions/commands/build/Commands.java

```diff
@@ -129,11 +130,37 @@ public static CommandData context(@Nonnull Command.Type type, @Nonnull String na
         return new CommandDataImpl(type, name);
     }
 
+    /**
+     * Create an activity entry point command builder.
+     *
+     * <p>Only one entry point can be created per app.
+     *
+     * @param  name
+     *         The entry point name, 1-32 lowercase alphanumeric characters
+     * @param  description
+     *         The entry point description, 1-100 characters
+     *
+     * @throws IllegalArgumentException
+     *         If any of the following requirements are not met
+     *         <ul>
+     *             <li>The name must be lowercase alphanumeric (with dash), 1-32 characters long</li>
+     *             <li>The description must be 1-100 characters long</li>
+     *         </ul>
+     *
+     * @return {@link EntryPointCommandData} builder for the app's activity entry point command
+     */
+    @Nonnull
+    public static EntryPointCommandData entryPoint(@Nonnull String name, @Nonnull String description)
+    {
+        return new EntryPointCommandDataImpl(name, description);
+    }
 
     /**
      * Parses the provided serialization back into an CommandData instance.
      * <br>This is the reverse function for {@link CommandData#toData()}.
      *
+     * <p><b>Note:</b> This does not take into account {@link #entryPoint(String, String) entry point commands}.
```

Comment:
Does this work for ContextCommands? If it works for ContextCommands, why not for EntryPoint commands?